### DISCUSSION TITLE
Additional Settings --> Custom Pages --> 404 Error Page

### DISCUSSION THREAD

**Discussion by d-floutier** [WebDevStudios/nextjs-wordpress-starter#243] [Q&A]
Hi guys !
Thx for the great work but 'im blocked at the beginnning of the install.
What do ya mean by that :
You'll need to change directories into your new WordPress install, remove wp-content, and clone the repo.

When i do it i got no more theme in my wordpress theme and i'm blocked ???

**Comment by d-floutier** [comment]
I manage to resolve my precedent question but now i can 't find Additional Settings --> Custom Pages --> 404 Error Page in my wordpress admin page ...

**Reply by gregrickaby** [reply]
At this time, you need Advanced Custom Fields Pro to view the options page. Is that installed an active?

**Reply by andrewreal** [reply]
Please could you mention this in the Wiki page, it stumped me until I found this thread.

**Reply by gregrickaby** [reply]
Hey Andrew,

It is mentioned. 

1. https://github.com/WebDevStudios/nextjs-wordpress-starter/wiki/Backend-Setup#dependencies
2. https://github.com/WebDevStudios/nextjs-wordpress-starter/wiki/Backend-Setup#step-7-set-page-options

Currently, @tommcfarlin is [working on creating an options page](https://github.com/WebDevStudios/nextjs-starter-wordpress/issues/10), so we no longer have to depend on ACF Pro.
